contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu

```diff
@@ -1,8 +1,8 @@
-id: pc-sanity-smoke-test
-_name: pc-sanity-smoke-test (Ubuntu Desktop)
+id: pc-sanity-base
+_name: pc-sanity-base
 unit: test plan
 _description:
-    This test plan is for smoke test as sanity before release image.
+    pc-sanity tests
```

Comment:
Can you add more description here? for example:

1. what are in the plan, who contribute it.
2. what is the plan for, who should include it or in what circumstance we should include it.